src/providers/lazy-loading/paramsToQuery.ts

```diff
@@ -47,8 +47,14 @@ export function filtersToQuery(
   filters: { [fieldName: string]: any }
 ): FireStoreQuery {
   const res = Object.entries(filters).reduce((acc,[fieldName, fieldValue]) => {
-    const opStr = fieldValue && Array.isArray(fieldValue) ? 'in' : '==';
-    return acc.where(fieldName, opStr, fieldValue);
+    if (Array.isArray(fieldValue)) {
+      return acc.where(fieldName, 'in', fieldValue);
+    } else {
+      return acc
+        .where(fieldName, '>=', fieldValue)
+        .where(fieldName, '<', fieldValue + 'z');
```

Comment:
This is a fairly significant change, it changes the firebase query from an exact match `==`, to a greater or equal than match, is that what we want? Will that work for text and numbers?

Reply:
Good question. Probably will not work as is with the numbers and need to add another case for number comparisons. 

This was the only way I could find that allows some sort of filtering for text with partial value

Reply:
Thanks for the explanation, maybe we could handle the strict filtering case and partial filtering cases seperately.

> This was the only way I could find that allows some sort of filtering for text with partial value

I had no idea that the `>=` and `<` good match partial strings! 😮 

Will probably merge #255 first, but can merge this after